File: trimming-random/5000/data/fizzbuzz/src/fizzbuzz.c
#include <stdio.h>

// FizzBuzzのロジック
void fizzBuzz(int n) {
    if (n % 15 == 0) {
        printf("FizzBuzz\n");
    } else if (n % 3 == 0) {
        printf("Fizz\n");
    } else if (n % 5 == 0) {
        printf("Buzz\n");
    } else {
        printf("%d\n", n);
    }
}

int main() {
    // 1から30まで実行
    for (int i = 1; i <= 30; i++) {
        fizzBuzz(i);
    }
    return 0;
}

File: trimming-middle/3000/data/fizzbuzz/src/fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

// FizzBuzzのロジック
func fizzBuzz(n int) string {
	if n%15 == 0 {
		return "FizzBuzz"
	} else if n%3 == 0 {
		return "Fizz"
	} else if n%5 == 0 {
		return "Buzz"
	}
	return strconv.Itoa(n)
}

func main() {
	// 1から30まで実行
	for i := 1; i <= 30; i++ {
		result := fizzBuzz(i)
		fmt.Println(result)
	}
}